cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -238,10 +239,20 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 		}
 	} else {
 		if isWCOW {
-			// The pause container activation will immediately exit on Windows
+			defaultArgs := "c:\\windows\\system32\\cmd.exe"
```

Comment:
Just to be sure, there isn't a case where just "cmd.exe" is passed in?

Reply:
+1

Reply:
We wanted to make sure that if someone is using a nanoserver image instead of a K8s pause image they wouldn't be broken after these changes.
I think `c:\windows\system32\cmd.exe` is the default cmd/entrypoint for the nanoserver image so if we see that in the image manifest we still want to launch `ping.exe` instead of running the default command of the pause image.

Reply:
@marosset I think github is having issues posting Kirtana's replies here. But the idea with adding "cmd.exe" was just in case someone was using nanoserver as their base image but for some reason replacing the entrypoint with just "cmd.exe" instead of the full path. I'm not sure if that's a scenario at all here but since this code will be removed in the near future, I figured it would be okay to include just in case. What are your thoughts?

Reply:
Had a discussion with Kevin about this today. There could be several different entrypoints that could break  like cmd.exe, windows\system32\cmd.exe etc and it would be an overkill to check for them. Moving it back to check for just c:\Windows\system32\cmd.exe which is the default entrypoint of nanoserver/servercore if nothing is touched